### ISSUE TITLE
You can get Scrounged Scythe as a card in your pool instead of Harvest Hand

### ISSUE THREAD

**Issue by drmDev** [magefree/mage#1698]
You can get Scrounged Scythe as a card in your pool instead of Harvest Hand

from mega thread
https://github.com/magefree/mage/issues/1676

Card is a double faced card. Harvest Hand is the day card and unimplemented. Scythe is the night side of the card but coded as if its an equipment of its own.

**Comment by drmDev** [comment]
Harvest Hand // Scrounged Scythe is being worked on at the moment. Issue will be resolved upon proper implementation of day/night cards here.

**Comment by halljared** [comment]
I have been working on it but I can't get it into decks as the scarecrow. I've chalked it up as a metadata issue that I'm unequipped to handle myself. I think the card mechanics I've coded are correct otherwise.

I wouldn't mind seeing transform reworked a bit personally though as it seems unwieldy to have separate "cards" as double faced cards. But it seems like a tall order to rework them at this point.
